optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/shift/ShiftRestServiceImpl.java

```diff
@@ -64,6 +65,17 @@
     @Inject
     private EmployeeRestService employeeRestService;
 
+    @SuppressWarnings("unused")
+    public ShiftRestServiceImpl() {
+    }
+
+    // For benchmark
+    public ShiftRestServiceImpl(EntityManager entityManager) {
```

Comment:
I 'd rather see the Generator refactor so it doesn't need the ShiftRestService to begin with.